### PR TITLE
throw errors when trying to use instagram but don't provide a token

### PR DESCRIPTION
**Pull request by JipSterk**
Implements #111

### CODE REVIEW

**Comment by raae** [comment]
I am forgetting how my own codes work and have not have time for a through look through the code. 
But I am thinking this might throw if one has no instagram links and no settings.

**Comment by raae** [comment]
Also thank you so much for contributing. I was a little tired this morning and forgot to be polite.

**Comment by JipSterk** [comment]
hi @raae its okay. we all get tired time to time its totally okay 🌈. i've never done anything with gatsby, but this would seem as a great plugin for a blog! also was very surprised with the amount of integrations currently. keep going and most importantly stay safe & stay sane ❤️

**Comment by raae** [comment]
@nickytonline did a "yarn format" on the project, if you could do that as well and I will approve.

**Comment by raae** [comment]
Seems we need to work on that formatting @nickytonline, not sure what is happening here.

**Comment by nickytonline** [comment]
@JipSterk, can you merge latest of master into your branch? It should contain all the updated formatting changes from prettier. I ran prettier on the entire code base in a previously merged PR.

Also @raae, I ran prettier directly on all the files in the code base. I wasn't aware you had a format command.

**Comment by nickytonline** [comment]
Hmm, it looks like when I ran prettier it resulted in different formatting than `npm run format` and I just discovered that `npm run format` does not actually change the files, it only outputs them. The issue is related to an extraneous `--`. Let me fix that up and then if you merge latest of master, it should all be good.

File: gatsby-remark-oembed-md-site/src/components/Bio.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
-import React from "react";
-import { Link } from "gatsby";
+import React from 'react'
```
Comment (nickytonline) [inline gatsby-remark-oembed-md-site/src/components/Bio.js:1]: In regards to the formatting, as I mentioned, `npm run format` wasn't persisting changes, but the changes here seem to be picking up a different prettier configuration as the project defaults to double quotes. Anyways, as I mentioned, I'll have a PR up to fix the formatting command, and then you can run it and things should be all good. There is work I'll be doing in the near future, to handle this on the pre-commit. See #120.

**Comment by raae** [comment]
I am sorry things are a little messy @JipSterk, I have marked this PR as hacktoberfest accepted in case you are participating in that and we do not get this closed by Sunday ;)

**Comment by nickytonline** [comment]
@JipSterk, thanks again for the PR.👏

If you merge latest from master, there should no longer be conflicts. If you have any issues doing this on your local,let me know and I can help sort it out.

**Pull request merged**